pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java

```diff
@@ -51,7 +51,7 @@
 /**
  * Generate metrics aggregated at the namespace level and optionally at a topic level and formats them out
  * in a text format suitable to be consumed by Prometheus.
- * Format specification can be found at {@link https://prometheus.io/docs/instrumenting/exposition_formats/}
+ * Format specification can be found at @see <a href="https://prometheus.io/docs/instrumenting/exposition_formats/">Exposition Formats</a>
```

Comment:
@see seems not working here.
<img width="720" alt="image" src="https://user-images.githubusercontent.com/16974619/160538096-d1c17571-c3ce-4824-8cd8-f12fad7c6213.png">
We need to place it in the new line to make it work. But I don't think we need `@see` here.

Reply:
I was reading about the `@see` and `@link` , and found that later is used for internal project classes, methods or variables. And for external links we should use `@see`. 
I will use only the href link, so, it will come like `See link`.